bedrock/firefox/tests/test_base.py

```diff
@@ -353,22 +355,22 @@ def test_fx_dev_browser_57_0_a2_whatsnew(self, render_mock):
         assert template == ["firefox/developer/whatsnew.html"]
 
     @override_settings(DEV=True)
-    @patch.dict(os.environ, SWITCH_DEV_WHATSNEW_68="False")
-    def test_fx_dev_browser_68_0_a2_whatsnew_off(self, render_mock):
```

Comment:
Note: the special whatsnew templates for 68 (aka Trailhead) were removed in https://github.com/mozilla/bedrock/pull/7665/ and the switch was removed in https://github.com/mozmeao/www-config/pull/430. The dev 68 test was redundant and safe to remove, but I ended up turning it into a test for the new page so the diff looks a little messy.